e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/keyboardNavigation.functional.ts

```diff
@@ -4629,3 +4629,62 @@ test('TreeList/DataGrid - Focus indicator is not visible when the Toolbar includ
   keyExpr: 'field_0',
   showBorders: true,
 }));
+
+test('Enter key should not trigger other function besides the function assigned on the clicked button', async (t) => {
+  const dataGrid = new DataGrid('#container');
+
+  await t
+    .click(dataGrid.getDataCell(0, 0).element)
+    .click(Selector('#editButton'))
+    .pressKey('tab')
+    .pressKey('enter');
+
+  const message = await Selector('#otherContainer').innerText;
+  const focusButtonMessage = message.includes(
+    'second button',
+  );
+
+  await t
+    .expect(focusButtonMessage)
+    .ok();
```

Comment:
Let's refactor this code block a little, I think there are two options:

1) Let's rename the ```focusButtonMessage```, because this name doesn't reflect the essence
(It's a boolean flag, not the focus button message)
E.g:
```suggestion
  const message = await Selector('#otherContainer').innerText;
  const isTextFromSecondBtn = message.includes('second button');

  await t
    .expect(isTextFromSecondBtn)
    .ok();
```

Or

2) Let's use the expect comparison :)
E.g:
```suggestion
  const message = await Selector('#otherContainer').innerText;

  await t
    .expect(message)
    .eql('second button', 'The message should be from the second button');
```